JASP-Desktop/po/jasp_nl.po

```diff
@@ -15498,6 +15470,175 @@ msgctxt "AnalysisFormExpander|"
 msgid "Remove this analysis"
 msgstr "Verwijder deze analyse"
 
+msgctxt "PrefsUI|"
+msgid "Disable Animations"
+msgstr "Schakel Animaties uit"
+
+msgctxt "PrefsUI|"
+msgid ""
+"Turns off all animations, this is implied when \"Safe Graphics Mode\" is on."
+msgstr ""
+"Schakelt alle animaties uit, dit is impliciet wanneer \"Veilige Grafische "
```

Comment:
is impliciet -> wordt geïmpliceert? Hmm... Ik weet het niet

Reply:
Impliciet lijkt me wel correct of misschien : 'dit is het geval wanneer'

Reply:
Nee dan beter impliciet ja